src/CheckoutProduct.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import './CheckoutProduct.css';
+import { useStateValue } from './StateProvider';
 
-const CheckoutProduct = ({ id, image, title, price, rating }) => {
+function CheckoutProduct({ id, image, title, price, rating }) {
+  const [{ basket }, dispatch] = useStateValue();
+
+  const removeFromBasket = () => {
+    dispatch({
+      type: 'REMOVE_FROM_BASKET',
+      id: id,
+    });
+  };
```

Comment:
here you could remove the brackets after fat arrow as and all code will be in 1 line.